StackExchange.Redis/StackExchange/Redis/RedisDatabase.cs

```diff
@@ -1579,6 +1579,605 @@ public IEnumerable<SortedSetEntry> SortedSetScan(RedisKey key, RedisValue patter
             return ExecuteAsync(msg, ResultProcessor.NullableDouble);
         }
 
+        public long StreamAcknowledge(RedisKey key, RedisValue groupName, string messageId, CommandFlags flags = CommandFlags.None)
```

Comment:
I think `messageId` (here and elsewhere - returns included) should be a `RedisValue` going in (for consistency as well), since we're converting underneath on every path used. Thoughts?